src/registrar/models/domain.py

```diff
@@ -1401,6 +1401,8 @@ def _disclose_fields(self, contact: PublicContact):
         DF = epp.DiscloseField
         fields = {DF.EMAIL}
         disclose = is_security and contact.email != PublicContact.get_default_security().email
+        # Delete after testing on other devices
```

Comment:
Will delete this once someone else tests the script and verifies that the discloses are set to True